modules/ppcp-api-client/src/Endpoint/OrderEndpoint.php

```diff
@@ -193,6 +193,10 @@ static function ( $item ) use ( &$contains_physical_goods ): bool {
 				: ApplicationContext::SHIPPING_PREFERENCE_GET_FROM_FILE
 			: ApplicationContext::SHIPPING_PREFERENCE_NO_SHIPPING;
 
+		if ( $this->items_without_shipping( $items ) ) {
+			$shipping_preferences = ApplicationContext::SHIPPING_PREFERENCE_NO_SHIPPING;
```

Comment:
What if there are some items with shipping and some without? Is it ok to have `SHIPPING_PREFERENCE_NO_SHIPPING` for the whole order then?